languagetool-language-modules/de/src/main/java/org/languagetool/rules/de/GermanSpellerRule.java

```diff
@@ -94,6 +94,7 @@ public class GermanSpellerRule extends CompoundAwareHunspellRule {
   private final Set<String> wordEndingsToBeProhibited   = new HashSet<>();
   private final Set<String> wordsNeedingInfixS          = new HashSet<>();
   private final Set<String> wordsWithoutInfixS          = new HashSet<>();
+  private final Set<String> germanPrefix         = new HashSet<>();
```

Comment:
Please call this germanPrefix**es** and indent it the same way as the line above

Reply:
fixed!